packages/dsh-tauri-pet/src/client/index.ts

```
@@ -0,0 +1,34 @@
+ *   注册进 layout 声明的 `shell.overlay`（list/root 槽，第三方通用浮层）——
+ *   右下角胶囊开关键 + 展开的「宠物」设置卡片：启用/停用、显示/隐藏、选择
+ *   桌宠（Codex/Deepseek），全部经 dsh-tauri invoke 桥调用桌面端 Tauri 命令
+ *   （get_pet_status/set_pet_enabled/set_active_pet/show_pet/hide_pet）。
+ *
+ * 依赖：slots（注册点位）、locale（双语文案）。invoke 桥来自 dsh-tauri/client。
+ */
+import type { ClientContext } from 'dsh-tauri/client'
+import { PET_CLIENT_PLUGIN } from './constants'
+import { installLocale } from './locales'
+import { mountPetStyles } from './styles'
+import { installPetControl } from './register/control'
```

ESLint

[error] Expected "./register/control" to come before "./styles".

(perfectionist/sort-imports)